### PR TITLE
Check the config value for core.ssl_verify

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

File: dcos/config.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def get_config_val(name, config=None):
     :param config: config
     :type config: Toml
     :returns: value of 'name' parameter
-    :rtype: object | None
+    :rtype: str | None
```
Comment (jsancio) [inline dcos/config.py]: I think this is correct. It should always return `str` or `None`, no?

Comment (tamarrow) [inline dcos/config.py]: yes

File: dcos/http.py [hunk 2]
```diff
@@ -44,9 +44,9 @@ def _verify_ssl(verify=None):
 
     if verify is None:
         verify = config.get_config_val("core.ssl_verify")
-        if verify.lower() == "true":
+        if verify and verify.lower() == "true":
```
Comment (tamarrow) [inline dcos/http.py]: this will fail if `verify` is a bool.

Comment (jsancio) [inline dcos/http.py]: If you believe the pydoc for `get_config_val` then at this point verify should be a `str` or `None`. In that case `verify` is true if `verify is not None and verify != ""`.

Comment (tamarrow) [inline dcos/http.py]: ah yes

**Pull request merged**